gensrc/proto/olap_file.proto

```diff
@@ -206,7 +206,8 @@ message TabletSchemaPB {
     optional double bf_fpp = 6; // OLAPHeaderMessage.bf_fpp
     optional uint32 next_column_unique_id = 7; // OLAPHeaderMessage.next_column_unique_id
     optional bool DEPRECATED_is_in_memory = 8 [default=false];
-    optional int64 id = 9;
+    optional int64 id = 9; // deprecated
+    optional int64 schema_id = 50;
```

Comment:
```suggestion
    optional int64 id = 50;
```

Reply:
> 

done, change in https://github.com/StarRocks/starrocks/pull/5314/commits/6a81340bbfff836c866b68f805fb80b5268e3e2c